packages/terra-alert/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Changed
+* Update tests to use `Terra.describeViewports` for setting viewports
```

Comment:
What caused the screenshot diffs?

Reply:
I believe the old code https://github.com/cerner/terra-core/pull/2498/files?short_path=debb75d#diff-9234219fdf2a8f3279af2bcab788b0d6L34 wasn't properly structured so the before hook in the above line wasn't being called and the viewport never got changed to tiny. The screenshots were newly generated.